.github/workflows/check.yml

```diff
@@ -0,0 +1,46 @@
+# Based upon:
+#   https://github.com/actions/starter-workflows/blob/main/ci/go.yml
+
+on:
+  pull_request:
+  push:
+    branches: [ $default-branch ]
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    strategy:
+      matrix:
+        go-version: [1.13, 1.14, 1.15]
+
```

Comment:
Not sure if we want this to be cross platform, but we can simply add here 
```
os: [macos-latest, ubuntu-latest, windows-latest]
```

Reply:
It's a valid question and equally applies to other repositories as we adopt workflows everywhere.

For the moment, however, I think focussing on the "most likely" host OS is acceptable. I'm going to keep this thought in the back of my mind, however, as I can imagine certain edge cases where perhaps Windows build and check might smoke out some issues around paths and the like.